config/locales/en.yml

```diff
@@ -694,7 +694,7 @@ en:
         study_mode: *course_path
         primary_course_subjects: *course_path
         iqts_country: edit_trainee_iqts_country_path
-        placement_detail: edit_trainee_placements_details_path
+        placement_detail: trainee_placements_confirm_path
```

Comment:
When do we use the edit details path now? (Just curious)

Reply:
It is for the blue notification on this page

![image](https://github.com/DFE-Digital/register-trainee-teachers/assets/470137/db604f3e-168b-4285-9a9c-1ca7266140a9)

taking the user to the edit page seems a bit wrong as they may of filled it in

![image](https://github.com/DFE-Digital/register-trainee-teachers/assets/470137/cf886c3f-47b9-4975-a4ae-b5646e0a53b0)